spec/02-integration/14-tracing/01-instrumentations_spec.lua

```diff
@@ -401,7 +415,7 @@ for _, strategy in helpers.each_strategy() do
 
         local spans = cjson.decode(res)
         assert_has_spans("kong", spans, 1)
-        assert_has_spans("kong.dns", spans, 2)
```

Comment:
Why in the old code there already were some spans with "kong.dns" ?

Reply:
because:
1. the first dns span is created when the postgres client establishes a connection with the DB, it will not be reported as it's not part of a request
2. the second dns span is created because there is a cosocket request [triggered by the tcp-trace-exporter](https://github.com/Kong/kong-ee/blob/0764e619be1289dde2c0eac891650151318f8796/spec/fixtures/custom_plugins/kong/plugins/tcp-trace-exporter/handler.lua#L40) plugin which is configured in this test
3. there was no third dns span because the upstream dns query was not recorded (which [this pr](https://github.com/Kong/kong/pull/11996) is fixing)